bowling/tests/bowlingGame.ut.cpp

```diff
@@ -62,3 +63,41 @@ SCENARIO("BowlingGame class can check if help is to be printed and trigger print
         }
     }
 }
+
+SCENARIO("Should save scores in file")
+{
+    GIVEN("Three arguments for start")
+    {
+        const char* userInput[] = {"./bowling", "../tests/testDirectory/goodInput/", "../tests/testDirectory/gameScores/result1.txt"};
+        BowlingGameStub bowlingGameStub(3, userInput);
```

Comment:
why do you use stub here? what is different in stub? You want to test your original BowlingGame. Stub can be used only when you need to pass it as a dependency to check some actions. But the real thing under test should be just BowlingGame here. And if you want to replace something with stub, it should inherit from the common interface as the real object. Stub should be removed.
```suggestion
        BowlingGame bowlingGame(3, userInput);
```

Reply:
Done :+1: